dipy/tracking/local/localtracking.py

```diff
@@ -93,12 +100,22 @@ def _generate_streamlines(self):
             directions = dg.initial_direction(s)
             directions = directions[:max_cross]
             for first_step in directions:
-                stepsF = local_tracker(dg, tc, s, first_step, vs, F, ss, fixed)
-                if stepsF < 0:
+                stepsF, tissue_class = local_tracker(dg, tc, s, first_step,
+                                                     vs, F, ss, fixed)
+                # enum TissueClass (tissue_classifier.pyd) is not accessible
```

Comment:
This seems to be the case in several locations here, where these values are hard-coded (e.g. tests). Worth putting up an issue on this, once we merge this PR